Lib/test/test_dis.py

```diff
@@ -2426,5 +2430,101 @@ def _unroll_caches_as_Instructions(instrs, show_caches=False):
                                   False, None, None, instr.positions)
 
 
+class TestDisCLI(unittest.TestCase):
+
+    def infile(self, content):
+        filename = tempfile.mktemp()
+        self.addCleanup(os_helper.unlink, filename)
+        with open(filename, 'w') as fp:
+            fp.write(content)
+        return filename
```

Comment:
Would be good to save this filename and add a `tearDown` method that deletes it when the test finishes.